README.md

```diff
@@ -46,24 +46,17 @@ You can clone the entire repo and work in the full project where we manage the [
 
 1. **Install Ruby Gems** - These gems are needed to be installed for the documentation site: `gem install ruby bundle jekyll`
 
-1. **Install Bundle Dependencies** - Bundle dependencies must be installed the from `docs` directory:
-`cd docs` and then `bundle install`
+1. **Serve the documentation website locally** - `npm run start:docs`
 
-1. **Serve it locally** - The documentation website is served from the `docs/_site` directory which is generated by Jekyll. To generate and launch the documentation site, from the root simply execute `npm start`
+1. **Serve the development playground locally**: `npm start`
 
 
 ### Project Dependencies
 The project has the following prerequisites and works best with the noted versions.
 
 * Git (for downloading this repo)
-* [Node LTS](https://nodejs.org/) 8.12.0
-* [Gulp](https://gulpjs.com/) 6.4.1
-* SCSS (for CSS)
-* Nunjucks (for testing)
-* Ruby (for documentation site)
+* [Node LTS](https://nodejs.org/)
 * Python 2 (for Jekyll - documentation site)
-* Bundle (for documentation site)
-* Jekyll (for documentation site)
 * Xcode (for some macOS users)
```

Comment:
i am all for cleaning up this list.. but is it right? seems like this list should be

* git
* npm/node
* Ruby
* python

I also think that ruby and pythons should be listed as peerdeps.

everything else is a dependency on one of those 4.

Reply:
I *think* that python can also be removed if we add 

`pygments: false` to `docs/_config.yml` this is based on 

https://github.com/jekyll/jekyll/issues/1131

if we do this, we might also be able to close #1300

Reply:
@bcullman `pygments` was replaced with `highlighter`, we're using `rouge`, not `pygments` so we should be fine to just nix the dependency.

Reply:
I don't think ruby should be a peer dep because it's not needed to _use_ this library, just for running the documentation site locally.